app/main.py

```
@@ -112,16 +121,28 @@ async def snapshot_builder(base_url: str) -> dict[str, object]:
             # Treat this as a normal shutdown path.
             pass
         finally:
+            try:
+                sync_service.stop()
+                if sync_task is not None:
+                    await asyncio.wait_for(sync_task, timeout=5)
+            except asyncio.TimeoutError:
+                if sync_task is not None:
+                    sync_task.cancel()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)